hazelcast/src/main/java/com/hazelcast/client/ClientEndpoint.java

```diff
@@ -44,6 +45,10 @@
 
     void setListenerRegistration(String service, String topic, String id);
 
+    void setListenerRemoveAction(String registrationId, Callable<Boolean> removeAction);
```

Comment:
What is this method supposed to do? From the naming it isn't clear to me and there is no javadoc.